config/v1/types_infrastructure.go

```diff
@@ -199,6 +203,26 @@ type OpenStackPlatformStatus struct {
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
 }
 
+// OvirtPlatformStatus holds the current status of the  oVirt infrastructure provider.
+type OvirtPlatformStatus struct {
+	// apiServerInternalIP is an IP address to contact the Kubernetes API server that can be used
+	// by components inside the cluster, like kubelets using the infrastructure rather
+	// than Kubernetes networking. It is the IP that the Infrastructure.status.apiServerInternalURI
+	// points to. It is the IP for a self-hosted load balancer in front of the API servers.
+	APIServerInternalIP string `json:"apiServerInternalIP,omitempty"`
```

Comment:
so these fields are required but `omitempty` ??

@sttts / @deads2k any guidance here?

Reply:
`omitempty` with string will cause they are not printed when empty, but it doesn't make any difference during parsing. I don't think we have clear guidance for it, though.

Reply:
omitempty for optional fields is often a good idea. Usually only if you want to distinguish undefined and zero value, you don't use omitempty.

Reply:
> https://github.com/openshift/api/blob/master/config/v1/doc.go#L5

all fields are kindof optional at package level, and this is similar to other structs, so following that seems fine to me.